charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -197,96 +217,87 @@ data:
+          # to cleanup evicted/failed/succeeded pods.
+          cleanup:
+            enabled: true
+            schedule: "*/15 * * * *"
+      {{- end }}
+
+          # Disable the postgres subchart. The platform pass credentials down
+          # from the platform configuration.
+          postgresql:
+            enabled: false
 
     {{- if .Values.global.authSidecar.enabled }}
         # enables nginx auth sidecar for airflow deployments
-        authSidecar:
-          enabled: true
-          repository: {{ .Values.global.authSidecar.repository }}
-          tag: {{ .Values.global.authSidecar.tag }}
-          port: {{ .Values.global.authSidecar.port }}
-          pullPolicy: {{ .Values.global.authSidecar.pullPolicy }}
-          annotations:
-            {{- range $key, $value := .Values.global.extraAnnotations}}
-            {{ $key }}: {{ $value | quote }}
-            {{- end }}
+        # TODO: auth sidecar support isn't done yet
+        #authSidecar:
+        #  enabled: true
+        #  repository: {{ .Values.global.authSidecar.repository }}
+        #  tag: {{ .Values.global.authSidecar.tag }}
+        #  port: {{ .Values.global.authSidecar.port }}
+        #  pullPolicy: {{ .Values.global.authSidecar.pullPolicy }}
+        #  annotations:
+        #    {{- range $key, $value := .Values.global.extraAnnotations}}
+        #    {{ $key }}: {{ $value | quote }}
+        #    {{- end }}
 
     {{- end }}
 
-    # If we're in single namespace mode, we will also
-    # skip the configuration for elasticsearch. Houston will fallback
-    # to its default modes, which are false/disabled.
-    # XXX: We may want to decouple these configurations from the
-    # singleNamespace config. At the moment, this is a convenient
-    # helper to require a minimal config change to run the platform in
-    # single namespace mode. Fluentd requires a ClusterRole to
-    # enrich logs with kubernetes metadata. In a single namespace environment,
-    # we're likely not going to get a ClusterRole from admins.
-    {{- if not .Values.global.singleNamespace }}
-      elasticsearch:
-        enabled: true
-        connection:
-          host: {{ printf "%s-elasticsearch-nginx.%s" .Release.Name .Release.Namespace }}
-          port: 9200
+      # If we're in single namespace mode, we will also
+      # skip the configuration for elasticsearch. Houston will fallback
+      # to its default modes, which are false/disabled.
+      # XXX: We may want to decouple these configurations from the
+      # singleNamespace config. At the moment, this is a convenient
+      # helper to require a minimal config change to run the platform in
+      # single namespace mode. Fluentd requires a ClusterRole to
+      # enrich logs with kubernetes metadata. In a single namespace environment,
+      # we're likely not going to get a ClusterRole from admins.
+      {{- if not .Values.global.singleNamespace }}
+        elasticsearch:
```

Comment:
There are a bunch of whitespace changes in here and I think this is wrong.

Reply:
Are there chart tests for these helm features?

Reply:
I added some tests so this isn't completely empty. https://github.com/astronomer/astronomer/pull/1135/commits/cc13da08f1ab8eef7cbc4282d1169a2580e8f8f6